os/drivers/lcd/ili9341.c

```diff
@@ -0,0 +1,1318 @@
+#endif
+		.runbuffer        = g_runbuffer0,
+		.orient           = ILI9341_IFACE0_ORIENT,
+		.pxfmt            = ILI9341_IFACE0_PXFMT,
+		.bpp              = ILI9341_IFACE0_BPP,
+		.power            = 0,
+#ifdef CONFIG_LCD_DMA_SUPPORT
+		.putdma			  = ili9341_putdma0,
+#endif
+	},
+#endif
+#ifdef CONFIG_LCD_ILI9341_IFACE1
+	{
+		.lcd              = 0,
+		.putrun           = ili9341_putrun1,
+#ifndef CONFIG_LCD_NOGETRUN
+		.getrun           = ili9341_getrun1,
+#endif
+		.runbuffer        = g_runbuffer1,
+		.orient           = ILI9341_IFACE1_ORIENT,
+		.pxfmt            = ILI9341_IFACE1_PXFMT,
+		.bpp              = ILI9341_IFACE1_BPP,
+		.power            = 0,
+#ifdef CONFIG_LCD_DMA_SUPPORT
+		.putdma 		  = ili9341_putdma1,
+#endif
+	},
+#endif
+};
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name:  ili9341_getxres
+ *
+ * Description:
+ *   Get horicontal resolution of the connected LCD driver depending on the
+ *   configured display orientation.
+ *
+ * Input Parameters:
+ *   dev   - Reference to private driver structure
+ *
+ * Returned Value:
+ *
+ *   Horicontal resolution
+ *
+ ****************************************************************************/
+
+static inline uint16_t ili9341_getxres(FAR struct ili9341_dev_s *dev)
+{
+	if (dev->orient == ILI9341_MADCTL_LANDSCAPE_PARAM1 ||
+		dev->orient == ILI9341_MADCTL_RLANDSCAPE_PARAM1) {
+		return ILI9341_YRES;
+	}
+
+	return ILI9341_XRES;
+}
+
+
+/****************************************************************************
+ * Name:  ili9341_getyres
+ *
+ * Description:
+ *   Get vertical resolution of the connected LCD driver depending on the
+ *   configured display orientation.
+ *
+ * Input Parameters:
+ *   dev   - Reference to private driver structure
+ *
+ * Returned Value:
+ *
+ *   Vertical resolution
+ *
+ ****************************************************************************/
+
+static inline uint16_t ili9341_getyres(FAR struct ili9341_dev_s *dev)
+{
+	if (dev->orient == ILI9341_MADCTL_LANDSCAPE_PARAM1 ||
+		dev->orient == ILI9341_MADCTL_RLANDSCAPE_PARAM1) {
+		return ILI9341_XRES;
+	}
+
+	return ILI9341_YRES;
+}
+
+
+/****************************************************************************
+ * Name:  ili9341_selectarea
+ *
+ * Description:
+ *   Select the active area for displaying pixel
+ *
+ * Input Parameters:
+ *   lcd       - Reference to private driver structure
+ *   x0        - Start x position
+ *   y0        - Start y position
+ *   x1        - End x position
+ *   y1        - End y position
+ *
+ ****************************************************************************/
+
+static void ili9341_selectarea(FAR struct ili9341_lcd_s *lcd,
+							   uint16_t x0, uint16_t y0, uint16_t x1, uint16_t y1)
```

Comment:
please check format here,  how to arrange more lines  parameter.

Reply:
seems no need to modify